src/Cardano/CoinSelection.hs

```diff
@@ -53,20 +53,20 @@ import GHC.Generics
 -- for all of the outputs, and a /remaining UTxO set/ from which all spent
 -- values have been removed.
 --
-newtype CoinSelectionAlgorithm m e = CoinSelectionAlgorithm
+newtype CoinSelectionAlgorithm o m e = CoinSelectionAlgorithm
```

Comment:
why not to use `i` instead of `o` ? `o` for me invokes `output` ;-)

Reply:
@paweljakubas wrote:
> why not to use `i` instead of `o` ? `o` for me invokes `output` ;-)

I've updated this to use a more complete set of variables:

| Type Variable | Meaning |
|--|--|
| **`i`** | a unique identifier for an _input_ (an unspent transaction output). |
| **`o`** | a unique identifier for an _output_ (an address). |
| **`u`** | a unique identifier for an _unspent transaction output_. |

For more information: see the summary of this PR.

Reply:
IMHO, it is much more clear now. and it reads better